### DISCUSSION TITLE
【预期的表现】为何全屏礼物或者背景动效会严重掉帧！

### DISCUSSION THREAD

**Discussion by mianma12138** [Tencent/libpag#2410] [Q&A] [answered]
小动效还行，半屏或者全屏，比如礼物，就会一卡一卡的。桌面播放器也会卡顿，没有mp4的流畅。有这么一个现象。

**Comment by mianma12138** [comment]
<img width="299" alt="b8f537f6e8e7aea5e8f5273cf6c233b" src="https://github.com/user-attachments/assets/8aceb389-17cc-42ff-9541-fd55a2719571">

**Comment by mianma12138** [comment]
爆量红色，就是卡顿的地方。

**Comment by kevingpqi123** [comment, marked as answer]
同 https://github.com/Tencent/libpag/discussions/2402
